### ISSUE TITLE
Moving after getting caugh by a mimics on Dungeon.

### ISSUE THREAD

**Issue by DANVXT** [DiscordDungeons/Bugs#683]
This bug is what makes weak player able to advance to the highest floor without having a serious injury by mimics, this happen when you open the chest ( it was mimic ) you saw that the reaction button has been removed by the bot, but...  DOES IT PREVENT YOU FROM USING THE REACTION AGAIN ? The answer is, no...  Somehow, when you got engaged by a mimics, a player can just continue the dungeon run by INSERTING THE REACTION by their own, idk if this is a bug or it was because that the dungeon is still under work in progress, but if this is a bug, please fix it, if this bug keep stay on the dungeon, then many player can actually able to reach to the highest floor by cheating...

**Comment by Sven65** [comment]
TL;DR You can add the movement reactions manually to a mimic encounter and it'll work.
